components/loginGrid.tsx

```diff
@@ -2,8 +2,8 @@ import InfoPanel from "../components/dashboard/InfoPanel";
 import JiraLoginLink from "../components/JiraLoginLink";
 import SlackLoginLink from "../components/SlackLoginLink";
 import GitHubLoginLink from "../components/GitHubLoginLink";
-import GitLabLoginLink from "../components/GitLabLoginLink";
-import BitbucketLoginLink from "../components/BitbucketLoginLink";
+// import GitLabLoginLink from "../components/GitLabLoginLink";
```

Comment:
No comments

Reply:
I thought "we will wanna reuse this later" but I'll take the suggestion